samples/sensor/vl53l1x/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Sample for VL53L1X time-of-flight sensor
+tests:
+  sample.sensor.vl53l1x:
+    build_only: true
+    depends_on: i2c vl53l1x
```

Comment:
Is there any in-tree board that supports vl53l1x?

Reply:
I don't think so

Reply:
If there are no in-tree boards with:
```yaml
supported:
  - vl53l1x
```
then this sample will never build in CI